Mention CF_ZONE_ID in environment variables (#710)
cloudflare/workers-docs: src/content/tooling/wrangler/configuration/_index.md

  ```

>   Note that providing authentication credentials through environment variables will override whatever credentials you configured 

Reviewer: ```suggestion
  Note environment variables will override whatever credentials you configured in `wrangler config` or in your `wrangler.toml`. 
```
